cloud/aws-functions/orphaned_vpcs.py

```diff
@@ -8,31 +8,39 @@
 from boto3.exceptions import Boto3Error
 
 
-def get_clusters_to_terminate(aws_region):
-    clusters_for_deletion = []
-    eks_client = boto3.client('eks', region_name=aws_region)
-    clusters = eks_client.list_clusters()['clusters']
-    if not clusters:
-        logging.info(f"There are no clusters in cloud")
-        sys.exit("There are no clusters in cloud")
-
-    for cluster in clusters:
-        cluster = eks_client.describe_cluster(name=cluster)
-        if 'delete-cluster-after-hours' not in cluster['cluster']['tags'].keys():
-            clusters_for_deletion.append(cluster)
-        else:
-            cluster_lifetime = float(cluster['cluster']['tags']['delete-cluster-after-hours'])
-            current_time = datetime.datetime.now().timestamp()
-            creation_time = datetime.datetime.strptime(str(cluster['cluster']['createdAt']),
-                                                       "%Y-%m-%d %H:%M:%S.%f%z").timestamp()
-            if (current_time - creation_time) / 3600 > cluster_lifetime:
-                clusters_for_deletion.append(cluster)
-
-    if not clusters_for_deletion:
-        logging.info(f"There are no clusters for deletion")
-        sys.exit("There are no clusters for deletion")
-    return clusters_for_deletion
+def isVpcToTerminate(vpc):
```

Comment:
is there some reason to use camel case here and snake case for other functions naming?

Reply:
no, I will fix it. It is mixed of python and go (